### DISCUSSION TITLE
Two horizontal scrollbar problem.

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#142] [Q&A] [answered]
Need to remove 1 scrollbar.

**Comment by montasim** [comment, marked as answer]
Use overflow-scroll in tailwind css

**Comment by olx24** [comment]
Test

**Comment by Preetiraj3697** [comment]
The reason you're getting the scrollbar in an odd place is because it is on the <div> rather than on the page. This is a result of using overflow:auto;.

overflow:auto; tells CSS that you want that particular <div> to get scrollbars (either horizontal or vertical) when it is too small to display all its content.

Therefore the direct answer to your question is No; you can't position the scrollbar elsewhere on the page, since it is attached to the <div>.
